src/Sarif.Multitool.FunctionalTests/TestData/EndColumnMustNotBeLessThanStartColumn/EndColumnLessThanStartColumnInRelatedLocation_Expected.sarif

```diff
@@ -21,65 +20,76 @@
-          },
-          "message": "Analyzing 'EndColumnLessThanStartColumnInRelatedLocation.sarif'...",
-          "level": "note",
-          "time": "2016-08-29T23:11:40.966Z"
-        }
-      ],
-      "rules": {
-        "SARIF1013": {
-          "id": "SARIF1013",
-          "name": "EndColumnMustNotBeLessThanStartColumn",
-          "shortDescription": "The \"endColumn\" property of a region object must not be less than the \"startColumn\" property.",
-          "fullDescription": "The \"endColumn\" property of a region object must not be less than the \"startColumn\" property.",
-          "messageFormats": {
-            "Default": "{0}: The value of the \"endColumn\" property is {1}, which is less than the value of the \"startColumn\" property, which is {2}."
-          },
-          "defaultLevel": "error",
-          "helpUri": "https://rawgit.com/sarif-standard/sarif-spec/master/Static%20Analysis%20Results%20Interchange%20Format%20(SARIF).html"
+      "resources": {
+        "rules": {
+          "SARIF1013": {
+            "id": "SARIF1013",
+            "name": {
+              "text": "EndColumnMustNotBeLessThanStartColumn"
+            },
+            "shortDescription": {
+              "text": "The \"endColumn\" property of a region object must not be less than the \"startColumn\" property."
+            },
+            "fullDescription": {
+              "text": "The \"endColumn\" property of a region object must not be less than the \"startColumn\" property."
+            },
+            "messageStrings": {
+              "Default": "{0}: The value of the \"endColumn\" property is {1}, which is less than the value of the \"startColumn\" property, which is {2}."
+            },
+            "richMessageStrings": {},
+            "helpUri": "http://docs.oasis-open.org/sarif/sarif/v2.0/csprd01/sarif-v2.0-csprd01.html"
+          }
         }
       },
       "files": {
-        "file:///G:/Code/sarif-sdk/src/Microsoft.CodeAnalysis.Sarif.Multitool/TestData/EndColumnMustNotBeLessThanStartColumn/EndColumnLessThanStartColumnInRelatedLocation.sarif": {
-          "mimeType": "application/octet-stream"
+        "file:///C:/Code/sarif-sdk/src/Sarif.Multitool.FunctionalTests/TestData/EndColumnMustNotBeLessThanStartColumn/EndColumnLessThanStartColumnInRelatedLocation.sarif": {
+          "mimeType": "application/sarif-json"
         }
       },
-      "invocation": {
-        "commandLine": "\"G:\\Code\\sarif-sdk\\bld\\bin\\SarifCli\\AnyCPU_Debug\\SarifCli.exe\"  validate G:\\Code\\sarif-sdk\\src\\Microsoft.CodeAnalysis.Sarif.Multitool\\TestData\\EndColumnMustNotBeLessThanStartColumn\\EndColumnLessThanStartColumnInRelatedLocation.sarif -o G:\\Code\\sarif-sdk\\src\\Microsoft.CodeAnalysis.Sarif.Multitool\\TestData\\EndColumnMustNotBeLessThanStartColumn\\EndColumnLessThanStartColumnInRelatedLocation_Expected.sarif -j G:\\Code\\sarif-sdk\\src\\Sarif\\Schemata\\Sarif.schema.json ",
```

Comment:
>commandLine [](start = 9, length = 11)

why did this invocation data go away? are you stripping it elsewhere (as some of our scripts do)? is the multitool not emitting it by default (reasonably).

Reply:
It moved up to become `invocations[]`.

---
In reply to: [205218670](https://github.com/Microsoft/sarif-sdk/pull/968#discussion_r205218670) [](ancestors = 205218670)